vars/artifactSetVersion.groovy

```diff
@@ -0,0 +1,138 @@
+import com.sap.piper.ConfigurationLoader
+import com.sap.piper.ConfigurationMerger
+import com.sap.piper.GitUtils
+import com.sap.piper.Utils
+import com.sap.piper.versioning.ArtifactVersioning
+
+import groovy.text.SimpleTemplateEngine
+
+def call(Map parameters = [:]) {
+
+    def stepName = 'artifactSetVersion'
+
+    handlePipelineStepErrors (stepName: stepName, stepParameters: parameters) {
+
+        def gitUtils = parameters.juStabGitUtils
+        if (gitUtils == null) {
+            gitUtils = new GitUtils()
+        }
+
+        //ToDo: exit if pending git changes are in the current repo? TBD!
```

Comment:
I did some tests but changes are not found. What usage of git diff --quiet would you suggest?

Reply:
from [git docu](https://git-scm.com/docs/git-diff#git-diff---quiet): 

>-quiet
>    Disable all output of the program. Implies --exit-code.

In case there are no changes, the return code is 0, otherwise != 0
There is no output in the console in this case.

Reply:
Even with a git add before I get 0 though ...

Reply:
If there was a git add before it is
```git diff --quiet HEAD```

But I do not see a ```git add .``` before (???)

Reply:
Done now ...

Reply:
see c75fb983ced4b74a9f37e4371fe4e295bd0de039